docs/huggingface/tgi-0.9.3.md

```diff
@@ -0,0 +1,42 @@
+| MAX_BATCH_TOTAL_TOKENS   |            16000            | ***IMPORTANT*** This is one critical control to allow maximum usage of the available hardware.This represents the total amount of potential tokens within a batch. When using padding (not recommended) this would be equivalent of `batch_size` * `max_total_tokens`.However in the non-padded (flash attention) version this can be much finer.For `max_batch_total_tokens=1000`, you could fit `10` queries of `total_tokens=100` or a single query of `1000` tokens.Overall this number should be the largest possible amount that fits the remaining memory (after the model is loaded). Since the actual memory overhead depends on other parameters like if you're using quantization, flash attention or the model implementation, text-generation-inference cannot infer this number automatically.                                                                                                                                                              |
+| MAX_WAITING_TOKENS       |             20              | This setting defines how many tokens can be passed before forcing the waiting queries to be put on the batch (if the size of the batch allows for it). New queries require 1 `prefill` forward, which is different from `decode` and therefore you need to pause the running batch in order to run `prefill` to create the correct values for the waiting queries to be able to join the batch.With a value too small, queries will always "steal" the compute to run `prefill` and running queries will be delayed by a lot.With a value too big, waiting queries could wait for a very long time before being allowed a slot in the running batch. If your server is busy that means that requests that could run in ~2s on an empty server could end up running in ~20s because the query had to wait for 18s.This number is expressed in number of tokens to make it a bit more "model" agnostic, but what should really matter is the overall latency for end users. |
+| HOSTNAME                 |            3000             | The IP address to listen on-p, --port <PORT>The port to listen on                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
This is the incorrect ENV Var for the description and default columns.  Seems to have gotten muddled with `PORT`.   `HOSTNAME` should have default of `0.0.0.0` and description of `The IP address to listen on.` The `PORT` ENV var should have default `3000` and description `The port to listen on`